src/screens/RequestSwap/SwapScreen.tsx

```diff
@@ -203,30 +322,31 @@ const styles = StyleSheet.create({
         height: '100%',
         width: '100%',
         alignItems: 'center',
+        backgroundColor: 'white',
     },
     filterContainer: {
-        height: '25%',
+        flex: 1,
         width: '100%',
         flexDirection: 'row',
         justifyContent: 'space-between',
         alignItems: 'center',
-        padding: 10,
+        padding: 5,
     },
     listContainer: {
-        flex: 1,
+        flex: 6,
         width: '100%',
         zIndex: -1,
         justifyContent: 'center',
         alignItems: 'center',
-        backgroundColor: 'rgb(108, 207, 212)',
-        borderTopLeftRadius: 20,
+        borderTopLeftRadius: 20, //TODO figure out why these dont work
         borderTopRightRadius: 20,
```

Comment:
alternatively, you could see if something like this works:
```suggestion
        borderRadius: '20px 20px 0 0',
```
(if you give `border-radius` 4 values: top-left, top-right, bottom-right, bottom-left (clockwise))
(need to specify units like `px` when you use shorthand multiple values in string)

Reply:
Unfortunately react-native doesn't let you pass a string into borderRadius. I think this has something to do with the arrangement of my containers but I havent dug into it yet